app/views/shared/summary_cards/_group.html.slim

```diff
@@ -1,38 +1,32 @@
 .summary-card
   .summary-card__body
-    .summary-card__row--half
-      h5.summary-card__title
-        span.glyphicon.glyphicon-th-large.glyphicon--right
-        = link_to resource.name, resource
-        - if defined?(hit)
-          .summary-card__relevancy
-            = render 'shared/components/score', hit: hit  
-    .clearfix
+    .summary-card__row--double
+      div class="entity-icon entity-icon--groups"
+        i class="fa fa-users"
+      .summary-card__title
+        h5= link_to resource.name, resource
+        h6
+          = humanize_date(resource.published_at)
+      - if defined?(hit) && hit
+        .summary-card__relevancy style="background-color: rgba(95, 125, 140, #{score_opacity(hit._score)});"
+          = "#{(hit._score).round(1)}"
+      .clearfix
     .summary-card__row
       p
-        strong ABOUT 
-        = resource.long_description
-    .summary-card__row
-      = render 'shared/components/tags_list', tags: resource.cached_tags
-    .summary-card__row.summary-card__row--last 
-      .summary-card__metadata
-        p
-          span.glyphicon.glyphicon-calendar.glyphicon--right
-          = humanize_date(resource.published_at)
-      .summary-card__metadata
-        p
-          span.glyphicon.glyphicon-comment.glyphicon--right
+        = resource.short_description
+    = render 'shared/components/tags_list', tags: resource.cached_tags
+    .summary-card__row.summary-card__row--last
+      .summary-card__more.summary-card__more--space.summary-card__more--buttons
+        = link_to 'Add to a list (25)', root_path, class: 'btn btn-gc'
```

Comment:
Not sure where the CSS resides but can you remove the shadow on this button: https://cl.ly/3F0J1n3n1Z3D